### ISSUE TITLE
Wrong reference #2

### ISSUE THREAD

**Issue by Elcomsoft** [usnistgov/mobile-threat-catalogue#286]
**References**:
- Here is the correct reference: 
[iOS 10: Security Weakness Discovered, Backup Passwords Much Easier to Break](https://blog.elcomsoft.com/2016/09/ios-10-security-weakness-discovered-backup-passwords-much-easier-to-break) by Oleg Afonin

**Comment by jajmo** [comment]
Which threat is this in reference to?

**Comment by Elcomsoft** [comment]
Exploitation of mobile device backups stored on a compromised PC
https://pages.nist.gov/mobile-threat-catalogue/ecosystem-threats/ECO-0.html
- in References

**Comment by jajmo** [comment]
I see, the reference title is wrong. Thanks for pointing this out!

**Comment by Elcomsoft** [comment]
Yes, the title and Oleg Afonin's name too it should be O. Afonin :)
